### PR TITLE
Fix for issue 2082 - problems when running tests in VS

### PR DESCRIPTION
**Pull request by lajones**
@ajcvickers Fix for #2082.

### CODE REVIEW

**Comment by ajcvickers** [comment]
:sheep::it:

File: test/EntityFramework.Commands.FunctionalTests/ExecutorTest.cs [hunk 1]
```diff
@@ -58,11 +58,13 @@ public SimpleProject()
                                     BuildReference.ByName("System.Interactive.Async", copyLocal: true),
                                     BuildReference.ByName("System.Runtime, Version=4.0.10.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"),
                                     BuildReference.ByName("System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089"),
+                                    BuildReference.ByName("System.Reflection.Metadata, Version=1.0.19.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a", copyLocal: true),
```
Comment (bricelam) [inline test/EntityFramework.Commands.FunctionalTests/ExecutorTest.cs]: Can we not put versions, public key tokens, etc. here?

Comment (bricelam) [inline test/EntityFramework.Commands.FunctionalTests/ExecutorTest.cs]: (Doing so will decouple it from package upgrades.)

Comment (lajones) [inline test/EntityFramework.Commands.FunctionalTests/ExecutorTest.cs]: Yep. It works without them. I've already committed to dev based on previous sign-off but I'll go through, change all the System.Reflection.Metadata references and re-commit.

Comment (bricelam) [inline test/EntityFramework.Commands.FunctionalTests/ExecutorTest.cs]: Cool. The framework ones (e.g. System.Runtime) for whatever reason needed to be full assembly names, but any that we just use the short name for, we should.

**Comment by lajones** [comment]
Fixes checked in with commits 3325df4 & 5bbd6e0.

**Pull request closed without merging**